src/platform/forms-system/src/js/web-component-patterns/relationshipToVeteranPattern.jsx

```diff
@@ -1,27 +1,22 @@
 import { radioUI, radioSchema } from './radioPattern';
 import VaTextInputField from '../web-component-fields/VaTextInputField';
 
-export const relationshipToVeteranUI = (
-  relationshipToVeteranTitle,
-  otherRelationshipToVeteranTitle,
-) => {
+export const relationshipToVeteranUI = personTitle => {
   return {
     relationshipToVeteran: radioUI({
-      title:
-        relationshipToVeteranTitle ??
-        'What’s your relationship to the veteran?',
+      title: `What’s your relationship to the ${personTitle ?? 'veteran'}?`,
```

Comment:
nit: since we use the same logic for all of these you could set it as a variable at the top and then reference it throughout
`const personTitleText = personTitle ?? 'veteran'`